manifests/g/Google/AndroidM3Emulator/m3-rc20a/Google.AndroidM3Emulator.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.AndroidM3Emulator
+PackageVersion: m3-rc20a
+Installers:
+- InstallerUrl: https://dl.google.com/android/archives/android_sdk_windows_m3-rc20a.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe

```

Comment:
```suggestion
  NestedInstallerType: portable
```

Reply:
When set to portable, this package runs from its local directory, but the symlink gives the error:
> Cannot locate system directory, pluse use -system <directory> to specify one

I think it needs to be run as `emulator -system %appdatalocal%\Microsoft\WinGet\Packages\Google.AndroidM3Emulator__DefaultSource\android_sdk_windows_m3-rc20a\tools\` but I'm unsure how to specify this argument for the symlink.